charts/openshift-metering/templates/reporting-operator/reporting-operator-config.yaml

```diff
@@ -17,6 +17,9 @@ data:
   enable-finalizers: {{ $operatorValues.spec.config.enableFinalizers | quote}}
 {{- if $operatorValues.spec.config.prestoTLS.enabled }}
   presto-ca-file: "/var/run/secrets/presto-tls/tls.crt"
+{{- if $operatorValues.spec.config.prestoAuth.enabled }}
+  presto-server-file: "/var/run/secrets/presto-auth/"
```

Comment:
This is a directory, but the reporting operator seems to be expecting a file at this location?

Reply:
The current implementation expects a directory, see `operator.go`:
```
serverCert, err := tls.LoadX509KeyPair(op.cfg.PrestoServerFile+"tls.crt", op.cfg.PrestoServerFile+"tls.key")
```

This could be removed and we could instead use separate variables for the cert/key location

Reply:
Yes please do that.